src/pages/news-announcements.tsx

```diff
@@ -112,11 +112,11 @@ NewsAnnouncements.getLayout = (page: React.ReactNode) =>
         navItems={[
           {
             path: '/',
-            label: <Breadcrumb>Service portal home</Breadcrumb>,
+            label: <span>Service portal home</span>,
           },
           {
             path: '/news-announcements',
-            label: <Breadcrumb>News & Announcements</Breadcrumb>,
+            label: <span>News & Announcements</span>,
```

Comment:
I actually think we don't need the `<span>` element as a replacement for `<Breadcrumb>`, since it gets injected into an `<li>`. 

Instead, we can wrap the string in single quotes and send it on its way. Is there any reason we need to keep the span?